src/schedule/ll_schedule.c

```diff
@@ -175,6 +207,9 @@ static void schedule_ll_tasks_execute(struct ll_schedule_data *sch)
 
 		tr_dbg(&ll_tr, "task %p %pU being started...", task, task->uid);
 
+#ifdef CONFIG_SCHEDULE_LOG_CYCLE_STATISTICS
+		cycles0 = (uint32_t)platform_timer_get(timer_get());
```

Comment:
We need to use the Zephyr APIs now for timer, it was merged a few days back. It defaults to the wrapper in xtos mode.